lib/html-import-template-bundler.js

```diff
@@ -32,25 +24,42 @@ export function bundle(cfg) {
         let content = fs.readFileSync(file, {
           encoding: 'utf8'
         });
-  
+
         let $ = whacko.load(content);
         let name = getCanonicalName(builder, file, 'view').replace(/!view$/g, '');
   
         $('template').attr('id', name);
         let template = $.html('template');
+
         templates.push(template);
       }
     });
 
-  fs.writeFileSync(outfile, templates.join('\n'));
+  let outFilePath = writeOutput(cfg.baseURL, cfg.bundleName, templates.join('\n'), cfg.options);
 
-  if (cfg.options.inject) {
-    injectLink(outfile, baseURL, cfg.options.inject);
+  if (cfg.options && cfg.options.inject) {
+    injectLink(outFilePath, baseURL, cfg.options.inject);
   }
 
   return Promise.resolve();
 }
 
+export function writeOutput(pathName, bundleName, output, options) {
+  let outFileName = utils.getOutFileName(output, bundleName + '.html', options && options.rev);
+  let outFilePath = path.resolve(pathName, outFileName);
+
+  if (fs.existsSync(outFilePath)) {
```

Comment:
I feel like the responsibility of `writeOutput` function should be just writing the output. This implementation adds another responsibility to it, which I believe should be in the controller function. e.g 

```javascript
function bundle() {
...
...
if (fs.existsSync(outFilePath)) {
    if (!cfg.force) {
      throw new Error(`A bundle named '${outFileName}' already exists. Use the --force option to overwrite it.`);
    }
    fs.unlinkSync(outFilePath);
  }

writeOutput(...);
...
...
}

This way the function doesn't have to return the filename. Again, computing file name is not it's responsibility either. We also can consider creating a new function that computes file name and another one that checks if file exists and remove it. Perhaps `removeIfExists(force)`